src/modules/70-pipeline/utils/pipelineReconcileUtils.ts

```diff
@@ -75,7 +87,16 @@ function hasDiffTemplateInputsAndStageOrPipeline<T extends { [key: string]: any
 
 function filterRuntimeInputs(obj: { [key: string]: unknown }, allPaths: string[], prefix?: string): string[] {
   return allPaths.filter(path => {
-    return `${get(obj, prefix ? `${prefix}${path}` : path)}`.startsWith('<+input>')
+    const fullPath = prefix ? `${prefix}${path}` : path
+    return `${get(obj, fullPath)}`.startsWith('<+input>') && !excludePath(fullPath)
+  })
+}
+
+function excludePath(path: string): boolean {
+  return PATHS_TO_EXCLUDE.some(pathToExclude => {
+    if (path.includes(pathToExclude)) {
+      return true
+    }
   })
```

Comment:
Not an blocker, but situations like this are perfect for implicit returns
```suggestion
  return PATHS_TO_EXCLUDE.some(pathToExclude => path.includes(pathToExclude))
```